packages/trader/src/Modules/Trading/Containers/trade-params-mobile.tsx

```diff
@@ -12,6 +10,62 @@ import React from 'react';
 import classNames from 'classnames';
 import { localize } from '@deriv/translations';
 
+type TTradeParamsModal = {
+    is_open: boolean;
+    toggleModal: () => void;
+};
+
+type TTextValueStrings = {
```

Comment:
i'll put this type into `common-prop.type.ts` and export. Just pull my branch and import.